app/controllers/edit-account.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+
+export default Ember.ObjectController.extend({
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw 'default'.